### PR TITLE
Search engine adapter fixes

### PR DESCRIPTION
**Pull request by cbielow**
[FIX] use Carbamidomethylation (fixed), Oxidation (variable) as default modifications for all relevant adapters.

[FIX] annotate default XTandem config in /SHARE/ of what values are actually defaults. Also switch off refinement by default.

[FIX] FileInfo now considers N/C-terminal peptide modifications in its summary

[FEATURE] sort modifications by case-INSENSITIVE names (much easier to find)

### CODE REVIEW

**Comment by cbielow** [comment]
StopWatch_test failed on Windows and Travis-clang timed out (even after restart).

any comments?

File: src/topp/FileInfo.cpp [hunk 1]
```diff
@@ -596,7 +596,7 @@ class TOPPFileInfo : public TOPPBase
             for (Size ia = 0; ia < aa.size(); ++ia)
             {
               if (aa[ia].isModified())
-                ++mod_counts[aa[ia].getModificationName()];
+                ++mod_counts[aa[ia].getModificationName() + "@" + aa[ia].getOneLetterCode()];
```
Comment (timosachsenberg) [inline src/topp/FileInfo.cpp]: getFullName besser?

**Comment by cbielow** [comment]
merge?

**Comment by timosachsenberg** [comment]
In principle, I am for it. 
I am just wondering about the N-term Acetylation. I think we want Protein N-term Acetylation. but currently only support peptide N-term. Smells like trouble ... ;)

**Comment by timosachsenberg** [comment]
+1 if passes

**Pull request merged**